src/Evolution/Executables/GeneralizedHarmonic/GeneralizedHarmonicBase.hpp

```diff
@@ -224,12 +224,16 @@ struct GeneralizedHarmonicTemplateBase<
     using factory_classes = tmpl::map<
         tmpl::pair<DenseTrigger, DenseTriggers::standard_dense_triggers>,
         tmpl::pair<DomainCreator<volume_dim>, domain_creators<volume_dim>>,
-        tmpl::pair<Event, tmpl::flatten<tmpl::list<
-                              Events::Completion,
-                              dg::Events::field_observations<
-                                  volume_dim, Tags::Time, observe_fields,
-                                  analytic_solution_fields>,
-                              Events::time_events<system>>>>,
+        tmpl::pair<
```

Comment:
don't make this change, just add `Events::ObserveNorms<norms_of_fields>,` to the list of Events

where you can above define `using norms_of_fields = tmpl::list<WHATEVER_FIELDS_YOU_WANT_NORMS_OF>;`